### DISCUSSION TITLE
是否考虑后期添加截图OCR识别功能

### DISCUSSION THREAD

**Discussion by zhangdingrui** [Snipaste/feedback#2908] [Ideas]
目前使用截图功能已经很完善。但是对于实际使用场景下，某个功能问题提供的截图需要识别功能中某个业务单据号，这种情况下还得使用windows10自带的识别功能，非常不方便。贵方可以考虑提供在专业版提供这个功能，这将会是一个亮点

**Comment by Geeyun-JY3** [comment]
https://github.com/Snipaste/feedback/issues/19#issuecomment-1846413616
